NEWS.md

```diff
@@ -1,5 +1,8 @@
 # random.cdisc.data 0.3.13.9060
 
+### Enhancements
+* Refactor `abab` parameters and visits to be align with `adpc`.
```

Comment:
Typo
```suggestion
* Refactor `abab` parameters and visits to align with `adpc`.
```

Reply:
Sure thing! These are the file locations within the package:

random.cdisc.data/inst/metadata/**ADAB.yml**
random.cdisc.data/vignettes/**rebuild_cached_data.Rmd**
random.cdisc.data/tests/testthat/**test-row_values.R**

Reply:
Thank you @edelarua. I run `test-row_values.R`, the message was below.

![Screen Shot 2023-05-04 at 12 37 31 PM](https://user-images.githubusercontent.com/24441440/236311682-0a07d57a-85f1-4a81-88b8-5ee97456dc36.png)

When I only run the adab part, the message was below.

```
── Warning (Line 2): radab produces expected values ─────────────────────
Following variables does not have label or are not found in metadata/adab.yml: VISIT, PARAM, PARCAT1, AVALC, AVAL, ISPTP, VISITDY, ARELTM1, NRELTM1, ARELTM2, NRELTM2, RELTMU, PARAMCD, AVALU, ADABLFL, ABLFL, ADABLPFL, ADPBLPFL
Backtrace:
 1. global radab(adsl, adpc, seed = 1)
 2. random.cdisc.data:::apply_metadata(adab, "metadata/adab.yml")
```

Is there anything else I should fix?

Thanks,
Qi

Reply:
I updated your branch to use the corrected version of `ADAB.yml` (should be uppercase) since we recently refactored this. Try pulling those changes and then running it again and the metadata should work.